stdlib/seq.mli

```diff
@@ -81,6 +81,14 @@ val iter : ('a -> unit) -> 'a t -> unit
     The traversal happens immediately and will not terminate on infinite
     sequences. *)
 
+val cons : 'a -> 'a t -> 'a t
+(** [cons x xs] returns a new sequence whose elements are [x] followed
+    by the elements of [xs] @since 4.11 *)
```

Comment:
The documentation is misleading: it is not a new sequence: `cons` potentially consumes its input `xs` sequence. Rewriting this as
```Ocaml
 [cons x xs] is the sequence containing the element [x] followed
    by the [xs] sequence @since 4.11 *)
```
would be more precise.